src/yarn/yarn.yml

```diff
@@ -150,5 +157,5 @@ jobs:
         type: string
         default: ""
     steps:
-      - run-release:
-          script: npx auto@${AUTO_VERSION:-<< parameters.version >>} shipit << parameters.args >> $AUTO_ARGS
+      - pre-release
+      - auto/shipit
```

Comment:
I'm assuming this is a 1-to-1 replacement to `run-release`?

Reply:
It is, yes. Previously we were calling `npx auto shipit` and that orb calls `auto shipit` where `auto` is the binary version of auto.